src-v2/NTS.Persistance/Adapters/SetupLapRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using NTS.Domain.Setup.Entities;
+using NTS.Persistence.Setup;
+
+namespace NTS.Persistence.Adapters;
+
+public class SetupLapRepository : SetRepository<Lap, SetupState>
+{
+    public SetupLapRepository(IStore<SetupState> store) : base(store)
+    {
+        state = store.Load().Result;
+    }
+
+    private SetupState state { get; set; }
+    public override Task<Lap> Update(Lap entity)
```

Comment:
As a general rule of thumb always use `async` with methods that return Task. [A good explanation why here](https://blog.stephencleary.com/2016/12/eliding-async-await.html)